examples/SoftwareSerial/platformio.ini

```diff
@@ -10,40 +10,39 @@
 
 [platformio]
 src_dir = ./
+lib_dir = ../../../
+default_envs = 
 
 [env]
 framework = arduino
 
 [env:uno]
-; ATmega328, 5V/16MHz
```

Comment:
Please leave the comments like this one, I find them useful to keep track of what each env support

Reply:
Unfortunately PlatformIO erases comments when you change a file from GUI. I hope they ever fix this [bug](https://github.com/platformio/platformio-core/issues/3862).

Reply:
> I can add back these comments at a later stage
You can use custom fields in environment sections. Just add custom_ prefix.

https://github.com/platformio/platformio-core/issues/3862#issuecomment-559966921

Reply:
Thanks, I'll do that whenever I came around to add the comments back